src/components/FeatureViews/Accounts/TabComponents/EnrollmentPayment.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Loading from "components/OmouComponents/Loading";
+import PaymentTable from "./PaymentTable";
+import {useQuery} from "@apollo/react-hooks";
+import Typography from "@material-ui/core/Typography";
+import gql from "graphql-tag";
+
+export const GET_ENROLLMENT_PAYMENTS = gql`
+query EnrollmentPayments($enrollmentId: ID!) {
+  enrollment(enrollmentId: $enrollmentId) {
+    paymentList {
+      __typename
+      accountBalance
+      createdAt
+      discountTotal
+      id
+      method
+      priceAdjustment
+      subTotal
+      total
+      updatedAt
+      registrationSet {
+        attendanceStartDate
+        createdAt
+        enrollment {
+          id
+        }
+        id
+        numSessions
+        updatedAt
+      }
+      enrollments {
+        id
+      }
+    }
+    __typename
+  }
+}
+`;
+
+const EnrollmentPayment = ({enrollmentID, courseID}) => {
+	const {data, loading, error} = useQuery(GET_ENROLLMENT_PAYMENTS,
+		{variables: {enrollmentId: enrollmentID}}
+	);
+    
+	if (loading) {
```

Comment:
Not really an issue but it makes your code less lines - you use conditionals without the brackets. 

```
if (loading) return <Loading/>
if (error) return <Typography>There's been an error! - {error.message}</Typography>
```

Reply:
Ok Great, made that update